manim/mobject/number_line.py

```diff
@@ -30,46 +32,46 @@ class NumberLine(Line):
 
     Parameters
     ----------
-    x_range : Union[:class:`list`, :class:`numpy.ndarray`]
+    x_range
         The :code:`[x_min, x_max, x_step]` values to create the line.
-    length : :class:`float`
+    length
         The length of the number line.
-    unit_size : class:`float`
+    unit_size
         The distance between each tick of the line.
-    include_ticks : :class:`bool`
+    include_ticks
         Determines whether ticks are included.
-    tick_size : :class:`float`
+    tick_size
         The vertical length of each tick mark.
-    numbers_with_elongated_ticks : Union[:class:`list`, :class:`numpy.ndarray`]
+    numbers_with_elongated_ticks
         A list of specific values with elongated ticks.
-    longer_tick_multiple : :class:`float`
+    longer_tick_multiple
         Influences how many times larger elongated ticks are than regular ticks (2 = 2x).
-    color : :class:`~.Colors`
+    color
         The color of the line.
-    rotation : :class:`float`
+    rotation
         The angle (in radians) at which the line is rotated.
-    stroke_width : :class:`float`
+    stroke_width
         The thickness of the line.
-    include_tip : :class:`bool`
+    include_tip
         Determines whether a tip is added to the end of the line.
-    tip_width : :class:`float`
+    tip_width
         The width of the tip.
-    tip_height : :class:`float`
+    tip_height
         The height of the tip.
-    include_numbers : :class:`bool`
+    include_numbers
         Determines whether numbers are added to tick marks. The number of decimal places is determined
         by the step size, this default can be overridden by ``decimal_number_config``.
-    label_direction : Union[:class:`list`, :class:`numpy.ndarray`]
+    font size
```

Comment:
```suggestion
    font_size
```

Reply:
The previous parameter, `number_scale_value` explicitly referenced numbers -- would it make sense to call this parameter `number_font_size`?

Reply:
I have a preference for just `font_size`. With the old name, it wouldn't be clear that it was the labels that were being scaled with just `scale_value`, whereas  font size is pretty synonymous with text.